tests/Directory.Build.props

```diff
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why remove this ?

Reply:
Its causes more problem than it fixes... Each test is should be and was always independently configured in their own `*.csproj`.

Reply:
I think that we should use the same sdk and libraries for all of them

Reply:
Can you apply this suggestion, @cschuchardt88 ? For us to move forward

Reply:
@cschuchardt88 What problems does it cause?